simapp/v2/testdata/config.toml

```diff
@@ -0,0 +1,598 @@
+# This is a TOML config file.
```

Comment:
is this needed?

Reply:
I found a setup example without needing the files. Good suggestion!